channel-api/src/main/java/com/synopsys/integration/alert/channel/api/issue/action/IssueTrackerTestAction.java

```diff
@@ -144,4 +149,9 @@ private ProjectIssueModel createPlaceholderProjectIssueModel(Long blackDuckConfi
         );
     }
 
+    //TODO: This addresses that hasErrors is set in the UI but this should be improved upon since it is not associated with a field
```

Comment:
Let's make sure this finds its way into a Jira ticket. "Provide a better mechanism for reporting errors from test-actions" or something like that.